javascript/ql/src/Security/CWE-834/TaintedLength.ql

```diff
@@ -0,0 +1,22 @@
+/**
+ * @name Tainted .length looping
+ * @description If server-side code iterates over a user-controlled object with
+ *              an arbitary .length value, then an attacker can trick the server
+ *              to loop infinitely. 
+ * @kind path-problem
+ * @problem.severity warning
+ * @id js/tainted-length-looping
+ * @tags security
+ * @precision low
```

Comment:
I think we can go all the way to `high` unless there are a lot of FPs. Otherwise `medium`.

Reply:
I expect a decent amount of FPs, so I put it to as `medium` for now.